packages/ui/lib/configs/guides.tsx

```diff
@@ -0,0 +1,135 @@
+import { Button, HeaderNav, Footer } from '..'
+
+const GUIDES_HEADER = {
+  extraClass: {
+    mobile: 'bg-[#fff7e9] px-6',
+  },
+  showSocialIcons: true,
+  actions: [
+    {
+      content: <Button variant="outlined-primary">Launch App</Button>,
+    },
+  ],
+  logo: {
+    url: '/',
+    domain: 'Guides',
+  },
+  menuSections: [
+    {
+      title: 'About Unlock',
+      url: '/about',
```

Comment:
Are you sure this relative URL will work from the Guides website?

Reply:
yes, guides is under `https://unlock-protocol.com/guides/` so this will redirec to `https://unlock-protocol.com/about`